tests/Storage/Device/AlibabaCloudTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Utopia\Tests\Storage\Device;
+
+use PHPUnit\Framework\TestCase;
+use OSS\Core\OssException;
+use Utopia\Storage\Device\Alibaba;
+
+class AlibabaCloudTest extends TestCase
+{
+        private $alibaba;
+    
+        protected function setUp(): void
+        {
+            $this->alibaba = new Alibaba('accessKey', 'secretKey', 'bucket', 'endpoint');
```

Comment:
You need to pass actual values of `accessKey`, `secretKey` etc. Those values can either be added to environment variables and read from there or you can take a reference from the S3 adapter.